src/tests/EdFi.DataManagementService.Tests.E2E/appsettings.json

```diff
@@ -1,25 +1,22 @@
 {
     "UseTestContainers": true,
     "Serilog": {
-        "Using": ["Serilog.Sinks.File", "Serilog.Sinks.Console"],
+        "Using": [ "Serilog.Sinks.File", "Serilog.Sinks.Console" ],
         "MinimumLevel": {
-            "Default": "Information"
+            "Default": "Verbose"
         },
         "WriteTo": [
+            { "Name": "Console" },
+            { "Name": "Debug" },
             {
                 "Name": "File",
                 "Args": {
                     "path": "./logs/.log",
                     "rollingInterval": "Day",
                     "outputTemplate": "{Timestamp:yyyy-MM-dd HH:mm:ss.fff} {Level:u3} {Message:lj}{NewLine}{Exception}"
                 }
-            },
-            {
-                "Name": "Console",
-                "Args": {
-                    "outputTemplate": "{Timestamp:yyyy-MM-dd HH:mm:ss.fff} {Level:u3} {Message:lj}{Exception}{NewLine}"
-                }
```

Comment:
@simpat-adam 
We are providing the format for the error information logging on the Console. Not sure about the default format. I guess, it is better to just keep this format.